frontend/pages/docs/directory.tsx

```diff
@@ -15,71 +15,30 @@ export interface DirectoryEntry {
 
 export const flatDirectory: Array<DirectoryEntry> = [
   // Main Docs
-  { title: 'Docs', slug: '', header: true },
+  { title: 'Overview', slug: '', header: true },
 
-  // Users
-  { title: 'Users', slug: 'users', header: true },
-
-  { title: 'Upload a Model', slug: 'users/upload-a-model', header: true },
-  { title: 'Why Upload A Model', slug: 'users/upload-a-model/why-upload-a-model' },
-  { title: 'Requirements', slug: 'users/upload-a-model/requirements' },
-  { title: 'Model Wrapper', slug: 'users/upload-a-model/model-wrapper' },
-  { title: 'Preparing The Code', slug: 'users/upload-a-model/preparing-the-code' },
-  { title: 'Upload To Bailo', slug: 'users/upload-a-model/upload-to-bailo' },
-  { title: 'Updating A Model', slug: 'users/upload-a-model/updating-a-model' },
+  { title: 'Getting Started', slug: 'getting-started', header: true },
 
-  { title: 'Deployments', slug: 'users/deployments', header: true },
-  { title: 'Requesting A Deployment', slug: 'users/deployments/requesting-a-deployment' },
-  { title: 'Compliance', slug: 'users/deployments/compliance' },
-  { title: 'Using A Deployment', slug: 'users/deployments/using-a-deployment', header: true },
-  { title: 'Docker', slug: 'users/deployments/using-a-deployment/docker' },
+  { title: 'FAQ', slug: 'getting-started/faq' },
+  { title: `What's new in V2`, slug: 'getting-started/whats-new-v2' },
```

Comment:
Title should have a question mark, single quotes instead of backquotes and the slug should probably be `getting-started/whats-new-in-v2`